src/l500/l535-options.cpp

```diff
@@ -0,0 +1,169 @@
+//// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
I would like each of the classes in this file to be in their own .cpp

Reply:
Or even in their own header, since the headers would all be included from within this .cpp

Reply:
done